dpsim/include/dpsim/DirectLinearSolver.h

```diff
@@ -57,5 +55,20 @@ namespace DPsim
 
 		/// solution function for a right hand side
 		virtual Matrix solve(Matrix& rightSideVector) = 0;
+
+		virtual void setConfiguration(DirectLinearSolverConfiguration& configuration)
+		{
+			m_configuration = configuration;
+			this->parseConfiguration();
+		}
+
+		protected:
+		/// Abstract linear solver class for MNA simulation
+		DirectLinearSolverConfiguration m_configuration;
+
+		virtual void parseConfiguration()
+		{
+			// no default parsing, configuration options vary for each solver
```

Comment:
Can we log a warning here so that it becomes clear that the user hands over a configuration which however is not processed by the specific `Adapter` class? For example in case of `SparseLUAdapter`, which does not override this method.

Reply:
logger included as discussed